### PR TITLE
Added sortby=topic query functionality

### PR DESCRIPTION
**Pull request by muxappo**

### CODE REVIEW

**Comment by Pipe-Smoking-Rabbit** [comment]
Hey Max, I think you forgot to add the test changes to the PR ? 🤔 
Also it seems as though you're adding `topic` as an _ORDER BY_ , but if this is for ticket 11 the idea is that it will get a **filtered** list of only articles for that topic 
Please resubmit for review

File: __tests__/integration.test.js [hunk 1]
```diff
@@ -174,6 +174,25 @@ describe("GET requests", () => {
...
+            expect(articles[0].topic).toBe("cats");
+          });
+      });
+
+      it("responds with 'No articles found' when given a topic that doesn't exist", () => {
+        return request(app)
+          .get("/api/articles?topic=dogs")
+          .then(({ body: { articles } }) => {
+            expect(articles).toBe("No articles found");
+          });
```
Comment (Poonam-raj) [inline __tests__/integration.test.js]: if the topic doesn't exist this more of a 404 status situation - it's a valid query but we don't have that topic, the message makes it seem like this is a topic that exists but that's not the case.

File: __tests__/integration.test.js [hunk 2]
```diff
@@ -174,6 +174,25 @@ describe("GET requests", () => {
...
+      });
+    });
```
Comment (Poonam-raj) [inline __tests__/integration.test.js:204]: please add a test checking that if the topic does exist but doesn't have any articles associated a valid response comes back

**Review by Poonam-raj — APPROVED** [review]
Just one more test needed and an amendment to the existing bad path. When you're done feel free to merge it in, I'm not too worried about how you implement the missing test but remember that a existing topic with no articles associated is a happy path test (the user has done nothing wrong, nothing bad has happened)

**Pull request merged**